pontoon/base/tests/test_models.py

```diff
@@ -0,0 +1,75 @@
+from django_nose.tools import assert_equal
+
+from pontoon.base.models import Translation
+from pontoon.base.tests import (
+    assert_redirects,
+    EntityFactory,
+    LocaleFactory,
+    ProjectFactory,
+    ResourceFactory,
+    TranslationFactory,
+    UserFactory,
+    TestCase
+)
+from pontoon.base.utils import aware_datetime
+
+
+class LocaleTests(TestCase):
+    def test_latest_activity_translated(self):
+        """
+        If latest activity in Translation QuerySet is translation submission,
+        return submission date and user.
+        """
+        user0 = UserFactory.create()
+        user1 = UserFactory.create()
+
+        translation0 = TranslationFactory.create(
```

Comment:
Also, you call `TranslationFactory.create` often enough for these tests that it might be worth it to create a helper to save on lines. Something with descriptive kwargs like this might work:

```python
latest_submission = self._translation(user0, submitted=(1970, 1, 1), approved=(1970, 1, 1))
```